src/frontend/screens/AuthScreen.tsx

```diff
@@ -74,53 +76,42 @@ export const AuthScreen = ({
     }
   }
 
-  return (
-    <ScrollView
-      ref={scrollViewRef}
-      style={{height: '100%', backgroundColor: WHITE}}>
-      <KeyboardAvoidingView behavior="padding" style={styles.container}>
-        <Image source={require('../images/icon_mapeo_pin.png')} />
-        <Text style={[styles.title]}>Mapeo</Text>
-
-        <Text style={[{marginBottom: 20, fontSize: 16}]}>
-          <FormattedMessage {...m.enterPass} />
-        </Text>
-
-        <PasscodeInput
-          error={error}
-          inputValue={inputtedPass}
-          onChangeTextWithValidation={setInputWithValidation}
-        />
+  const {top} = useSafeAreaInsets();
 
-        {error && (
-          <Text style={[styles.wrongPass]}>
-            <FormattedMessage {...m.wrongPass} />
-          </Text>
-        )}
-      </KeyboardAvoidingView>
-    </ScrollView>
+  return (
+    <ScreenContentWithDock
+      contentContainerStyle={[styles.contentContainer, {paddingTop: top}]}
```

Comment:
There doesn't seem to be any padding at the top. Can't see it in this screenshot, but it is right up against the bevel

<img width="293" alt="image" src="https://github.com/user-attachments/assets/2fa2e4eb-4d35-436f-9fd1-b50c33a5d353">

Reply:
addressed via https://github.com/digidem/comapeo-mobile/pull/503/commits/3cdbf5089f64437e9956714a8dbf553974a42bd7